gradle/deploy.gradle

```diff
@@ -37,7 +37,9 @@ afterEvaluate {
                 from components.release
                 artifactId libraryArtifactId
                 groupId = 'live.ditto'
-                version = findProperty("LIBRARY_VERSION")
+
+                def versionPropertyName = "LIBRARY_VERSION"
```

Comment:
I'm wondering if no LIBRARY_VERSION can be found I just set to "0.0.0" as a default?

Reply:
It means one less step in publishing locally.

Reply:
That's a good shout - maybe just use a different string to avoid potential confusion with real semver? `SNAPSHOT` or something like that. 

If you do it also keep in mind the order of precedence for properties - https://docs.gradle.org/current/userguide/build_environment.html#priority_for_configurations

Currently the pipeline injects the prop as an environment variable which has the lowest precedence according to their docs - so if you stick the hardcoded test version in `gradle.properties` it will always take it from there.

Reply:
I think that should work!

Reply:
Cool, I'll update the PR and readme